terraform/workspace-variables/review.tfvars

```diff
@@ -0,0 +1,27 @@
+# PaaS
+#paas_sso_passcode                = ""
+#paas_app_environment             = "qa"
```

Comment:
Can you replace comment with something along the lines of
```
#paas_app_environment not specified here, must be passed to terraform dymamically
```

Reply:
done